releasenotes/notes/adjust-ceph-permissions-on-images-for-cinder-user-3154b995916ee27e.yaml

```
@@ -0,0 +1,10 @@
+---
+fixes:
+  - |
+    Adjusts the documented permissions for the images pool for the cinder user.
+    This is to support copy on write snaphots of vms using ceph backed
+    ephemeral storage. Existing deployments will need to manually adjust these
+    values using the CLI (the collection is currently not smart enough to do this):
+    ``ceph auth caps client.cinder mon 'profile rbd' osd 'profile rbd
+    pool=volumes, profile rbd pool=vms, profile rbd pool=images' mgr 'profile
+    rbd pool=volumes, profile rbd pool=vms'``.
```

Couple of very minor nits, otherwise LGTM but someone with more Ceph knowledge than me should review before we merge
```suggestion
    Adjusts the documented permissions for the images pool for the Cinder user.
    This is to support copy on write snapshots of VMs using Ceph backed
    ephemeral storage. Existing deployments will need to manually adjust these
    values using the CLI (the collection is currently not smart enough to do this):
    ``ceph auth caps client.cinder mon 'profile rbd' osd 'profile rbd
    pool=volumes, profile rbd pool=vms, profile rbd pool=images' mgr 'profile
    rbd pool=volumes, profile rbd pool=vms'``.
```